app/models/concerns/searchable.rb

```diff
@@ -39,7 +39,7 @@ def refresh_index
     end
 
     # class method shorthand, useful for triggering via rake task or console or tests
-    def self.rebuild_index
+    def self.rebuild_index(_opts = {})
```

Comment:
The `say_when` cron jobs always pass an opts object.  Whether you want it or not.